src/pint/observatory/observatories.py

```diff
@@ -1,977 +1,86 @@
-"""Definitions for standard observatories.
+"""Standard observatories read in from observatories.json
```

Comment:
Should this whole file be gone?

Reply:
If the functions are already in `topo_obs.py` we definitely don't want them here too. It's easy enough to get it back - this is git - if we decide we want the functions here instead.